.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-issue-stale: 30
+          days-before-pr-stale: 30
+          days-before-close: 7
+          stale-issue-message: 'This issue has not had any updates in 30 days. If there are no updates/comments within 7 days, this issue will be closed.'
+          stale-pr-message: 'This pull request has not had any updates in 30 days. If there are no updates/comments within 7 days, this issue will be closed.'
+          close-issue-message: 'This issue has been closed due to inactivity. If your issue has not been resolved, please open a new issue and link this issue.'
+          close-pr-message: 'This issue has been closed due to inactivity. Please open a new pull request if needed.'
+          stale-issue-label: 'Stale'
+          close-issue-label: 'Auto-Closed'
+          stale-pr-label: 'Stale'
+          close-pr-label: 'Auto-Closed'
+          exempt-issue-labels: 'Browser Bug,Bug,dependencies,Documentation,feature-request,Triaged'
```

Comment:
Can we just do only auto close `Need More Information` for now?

Reply:
mm not so sure about that - the `Needs more info` label isnt removed from the issue when a customer replies, so we might have an issue where we mark the label, the customer replies, but the label is still on the issue. Then this tool might close the ticket if we dont reply within the specified time.